src/config/dark-sites.config

```diff
@@ -557,6 +559,7 @@ oldfag.org
 omcar.pl
 omgwtfnzbs.me
 onionplay.co
+onlix.me
```

Comment:
This isn't a dark by default site, please remove:

![image](https://user-images.githubusercontent.com/25481501/217864204-398283de-f01e-455a-a533-7864b9b1edd8.png)